validation/build_validation_databases.R

```diff
@@ -0,0 +1,63 @@
+# To use this file, run it as an R script in the validation directory
+# Ensure that the output databse is set to "ref" if building reference DBs, or "test" if building test DBs
+
+tool_directory <- "~/UNSW/MATCH/DER_disturbance_analysis"
+source(sprintf("%s/BDInterface/interface.R", tool_directory))
+library("logging")
+library("rjson")
+logging::basicConfig()
+
+data_dirs <- list.dirs('data', recursive=FALSE)
+output_database <- "ref" # "test"
+required_file_names <- c("ref_circuit_details.csv", "ref_meta_data.json", "ref_raw_data.csv", "ref_site_details.csv")
```

Comment:
In order to generate a set of reference data you'll need to run the tool at least once in order to get a representative sample of outputs, determine the pre event interval etc.
The reference data is based on the outputs in the first place, so I think it should be required for now. We could consider it once the tool is automated enough to generate those outputs, but I think the results should be manually inspected anyways so probably best to continue to require it.

Reply:
I just noticed that this script doesn't create a 'test_meta_data.json' to use when running the tool. Should it copy the ref file and save it with the test prefix?

Reply:
The meta data should be the same for both the test and ref runs since we want to compare with the same settings.
The only context where it would need to be separate is when the test branch has added new settings, in which case copying the ref file probably wouldn't work.
Do you think I should change the name from `ref_meta_data.json` to `common_meta_data.json` or something to make it more obvious that the same settings should be used?

Reply:
I guess it's ok, but it does mean that the database file path is incorrect for the test. I also think that the metadata file is forwards compatible, and any missing entries are added in the next time it is saved, so I would tend towards copying the file

Reply:
Good point. I'll copy the ref file as you've suggested, thanks!